src/views/ModelPlan/TaskList/ITSolutions/SolutionImplementation/_components/Solution.tsx

```diff
@@ -0,0 +1,173 @@
+import ImplementationStatuses from '../../_components/ImplementationStatus';
+import SolutionCard from '../../_components/SolutionCard';
+
+type flatErrorsType = {
+  [key: string]: string;
+};
+
+type SolutionTypes = {
+  solution: OpertionalNeedSolutionTypes;
+  // flatErrors: [key: string]: string;
+  identifier: string;
+  index: number;
+  handleOnBlur: (e: React.ChangeEvent<HTMLInputElement>, field: string) => void;
+  setFieldValue: (
+    field: string,
+    value: any,
+    shouldValidate?: boolean | undefined
+  ) => void;
+  isUpdatingStatus: boolean;
+  operationalNeed: OperationNeedType;
+  values: OperationNeedType;
+  flatErrors: flatErrorsType;
+  loading: boolean;
+};
+
+const Solution = ({
+  solution,
+  identifier,
+  index,
+  handleOnBlur,
+  setFieldValue,
+  isUpdatingStatus,
+  operationalNeed,
+  values,
+  flatErrors,
+  loading
+}: SolutionTypes) => {
+  const { t } = useTranslation('itSolutions');
+  const { t: h } = useTranslation('draftModelPlan');
+
+  return (
+    <div key={solution.id}>
+      <p className="text-bold">{t('solution')}</p>
+
+      <SolutionCard solution={solution} shadow />
+
+      {!loading && (
+        <>
+          <FieldGroup
+            scrollElement="mustStartDts"
+            error={!!flatErrors.mustStartDts}
+            className="margin-top-1"
+          >
+            <Label
+              htmlFor={`solution-must-start-${identifier}`}
+              className="text-bold"
+            >
+              {t('mustStartBy')}
+            </Label>
+
+            <div className="usa-hint margin-top-1">{h('datePlaceholder')}</div>
+
+            <FieldErrorMsg>{flatErrors.mustStartDts}</FieldErrorMsg>
+
+            <div className="width-card-lg position-relative">
+              <Field
+                as={DatePicker}
+                error={!!flatErrors.mustStartDts}
+                id={`solution-must-start-${identifier}`}
+                data-testid={`solution-must-start-${identifier}`}
+                maxLength={50}
+                name={`solutions[${index}].mustStartDts`}
+                defaultValue={solution.mustStartDts}
+                onBlur={(e: React.ChangeEvent<HTMLInputElement>) => {
+                  handleOnBlur(e, `solutions[${index}].mustStartDts`);
+                }}
+              />
+            </div>
+          </FieldGroup>
+
+          <FieldGroup
+            scrollElement="mustFinishDts"
+            error={!!flatErrors.mustFinishDts}
+          >
+            <Label
+              htmlFor={`solution-must-finish-${identifier}`}
+              className="text-bold"
+            >
+              {t('mustFinishBy')}
+            </Label>
+
+            <div className="usa-hint margin-top-1">{h('datePlaceholder')}</div>
+
+            <FieldErrorMsg>{flatErrors.mustFinishDts}</FieldErrorMsg>
+
+            <div className="width-card-lg position-relative">
+              <Field
+                as={DatePicker}
+                error={!!flatErrors.mustFinishDts}
```

Comment:
For both of these date errors, browser console throwing warning about an incorrect type.  The boolean here sohuld be expressed as:

```
error={+!!flatErrors.mustFinishDts}
```